compiler/msgs.nim

```diff
@@ -1032,7 +1039,7 @@ proc liMessage(info: TLineInfo, msg: TMsgKind, arg: string,
                          KindColor, `%`(KindFormat, kind))
       else:
         styledMsgWriteln(styleBright, x, resetStyle, color, title, resetStyle, s)
-      if msg in errMin..errMax and hintSource in gNotes:
+      if hintSource in gNotes:
```

Comment:
This is now the only line that has any impact.

Reply:
not sure why you closed this; the point was to allow showing line contexts for every entry; as I wrote earlier (and also mentioned by @alehander42 here https://github.com/nim-lang/Nim/issues/7586#issuecomment-381096369), currently nim only shows line contexts for errors (which in practice would be the last entry), skipping all the contexts for warnings

Reply:
I applied this change manually.